common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4144,6 +4183,12 @@
     ],
     "sqlState" : "42K05"
   },
+  "RESOLUTION_VALIDATION_ERROR" : {
+    "message" : [
+      "The analysis phase failed with an internal error. Reason: <message>"
```

Comment:
Shall we reuse the error class `INTERNAL_ERROR` for it?

Reply:
Sure, why not.